azerothcore/azerothcore-wotlk
# fix(DB/quest_offer_reward): Correct incorrect newline for Kranal Fiss

## Changes Proposed:
-  Changes a `$bb` to `$b$b` in quest_offer_reward entry for Call of Fire.

## Issues Addressed:

- Closes #13051 

## SOURCE:

[Wow Classic](https://youtu.be/O5j5GB0cX2A?t=62)

## Tests Performed:

- Untested.


## How to Test the Changes:


1. Run the query.
2. `.quest add 2984`
3. Go to [Kranal Fiss](https://www.wowhead.com/wotlk/npc=5907)
4. Observe the (hopefully) corrected text.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
